scripts/bootstrap.sh

```diff
@@ -150,6 +150,14 @@ fork_repos () {
 
 }
 
+check_infra () {
+   if [[ "${ADD_INFRA}" == "yes" ]]; then 
+     pushd ${OUTPUT_DIR}/gitops-0-bootstrap
+       bash ./scripts/infra-mod.sh
```

Comment:
Using `bash` does the environment variables get propagated to the infra-mod.sh script?

Reply:
I think we should consistency and use `source`
we already do in other places in the same script
https://github.com/cloud-native-toolkit/multi-tenancy-gitops/blob/master/scripts/bootstrap.sh#L352

Reply:
Updated to use `source`